www-nginx/ngx-brotli/ngx-brotli-20231109-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2025 Gentoo Authors
```

Comment:
Given there's only a _single_ tag in the upstream repo called `1.0.0_rc1` or whatever, we should treat every commit as _before_ the first release (as none exists), i.e. we should really version as 0_pre20231109.

Reply:
Probably best to just fix that now (and drop old immediately) before we stable. Please check the other nginx packages for this issue and handle that too if appropriate.

Note that if a tag _does_ exist and you're taking a snapshot after it, it should be LAST_TAG_pXXXX where XXXX is the date of last commit instead.

Reply:
It should be 0_pXXXX if there's never been any release, as `0` is a valid version to release. But you're right, there's already one tag, so it should be `1.0.0_rc_p20231109`, as it's a snapshot _after_ `1.0.0_rc`.

Reply:
Removing the current version means that on a world upgrade, it'll downgrade, as long as something pulled that package in to begin with.